stats/server/server_stats.go

```diff
@@ -95,3 +95,13 @@ type ResourceQueued struct {
 }
 
 func (s *ResourceQueued) isServerEvent() {}
+
+// IRVMatchedResource contains the stats for resources which has the same version as the one present in client.
```

Comment:
Can you be a bit more explicit here? This is only passed to the handler when a resource is not sent by the server because its version matches the one present in the request's initial_resource_versions